src/Core/Mesh/TopologicalTriMesh/TopologicalMesh.cpp

```diff
@@ -226,7 +220,7 @@ TriangleMesh TopologicalMesh::toTriangleMesh() {
         int i = 0;
 
         // iterator over vertex (thru halfedge to get access to halfedge normals)
-        for ( TopologicalMesh::FaceHalfedgeIter fh_it = fh_iter( *f_it ); fh_it.is_valid();
+        for ( TopologicalMesh::ConstFaceHalfedgeIter fh_it = cfh_iter( *f_it ); fh_it.is_valid();
```

Comment:
Nice to improve semantic (though it's the same)! Should also be applied within the Subdividers and the TopologicalMeshTest!

Reply:
Actually I had to change to Const because of the constness of toTriangleMesh member function. 
Feel free to improve semantic elsewhere if appropriate

Reply:
Resolved here, but could you please do the same for subdividers (I prefer not to mess up there ;))